sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_simulator.py

```diff
@@ -25,7 +25,7 @@
 from ._utils import JsonLineChatProtocol
 
 
-USER_AGENT += " (type=simulator subtype=Simulator)"
+USER_AGENT += " (type=simulator; subtype=Simulator)"
```

Comment:
Follow the standard UA format; use a semicolon to separate each key-value pair as the values may contain white spaces in the future.